assets/javascripts/bbcode-parser.min.js

```diff
@@ -51,20 +51,37 @@
     * @file Adds [highlight] to bbcode
     * @example [highlight]content[/highlight]
     */
-    const toNode = (tag, attrs, content) => ({
+    const toNode$1 = (tag, attrs, content) => ({
```

Comment:
We should probably consolidate these functions into a helper function somewhere to reduce code duplication across each tag.

Reply:
added https://github.com/RpNation/bbcode/issues/15